poleno/utils/forms.py

```diff
@@ -183,6 +183,16 @@ def render(self, name, value, attrs=None):
         return format_html(u'<span{0}>{1}</span><input{2} />',
                 flatatt(span_attrs), force_text(value), flatatt(input_attrs))
 
+class RangeWidget(forms.Widget):
+    def render(self, name, value, attrs=None):
+        if value is None:
+            value = u''
+        input_attrs = self.build_attrs(attrs, type=u'range', name=name, value=force_text(value))
+        return mark_safe(format_html(
+                u'<div class="pln-range-widget"><input{0} /><span></span></div>',
```

Comment:
@martinmacko47 takato html sablona je OK? Nebude potreba nastavovat attributy pre `div` a `span`. Ake su moznosti riesenia?

Reply:
Neviem. Bude? Ak ano, mozeme do `RangeWidget.__init__` pridat argumenty `div_attrs` a `span_attrs`, ulozit ich tam do `self.div_attrs` a `self.span_attrs`, a potom v `render` pouzit. Vid ako je to v `ObligeeWidget` pre tamojsie `input_attrs`.

Ale ak netreba, tak zbytocne. Pridame, ked bude treba.

Reply:
v tejto situacii netreba, takze neprepisujem,
:ok: